### ISSUE TITLE
Right clicked on the console title bar

### ISSUE THREAD

**Issue by furtadorebeca** [spyder-ide/spyder#15218]
### What steps will reproduce the problem?

Right clicked on the console title bar and received this error. 

### Traceback
```python-traceback
Traceback (most recent call last):
  File "/Applications/Spyder.app/Contents/Resources/lib/python3.9/spyder/plugins/ipythonconsole/plugin.py", line 977, in create_new_client
    self.add_tab(client, name=client.get_name(), filename=filename)
  File "/Applications/Spyder.app/Contents/Resources/lib/python3.9/spyder/plugins/ipythonconsole/plugin.py", line 1567, in add_tab
    self.switch_to_plugin()
  File "/Applications/Spyder.app/Contents/Resources/lib/python3.9/spyder/api/plugins.py", line 406, in switch_to_plugin
    super(BasePluginWidget, self)._switch_to_plugin()
  File "/Applications/Spyder.app/Contents/Resources/lib/python3.9/spyder/plugins/base.py", line 284, in _switch_to_plugin
    self.main.maximize_dockwidget()
  File "/Applications/Spyder.app/Contents/Resources/lib/python3.9/spyder/app/mainwindow.py", line 1164, in __getattr__
    return super().__getattr__(attr)
AttributeError: 'MainWindow' object has no attribute 'maximize_dockwidget'
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.9.2
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Darwin 20.3.0

### Dependencies

```

# Mandatory:
applaunchservices >=0.1.7     :  0.2.1 (OK)
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.0.1 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
setuptools >=39.0.0           :  54.1.2 (OK)
sphinx >=0.6.6                :  3.5.3 (OK)
spyder_kernels >=2.0.1,<2.1.0 :  2.0.1 (NOK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  22.0.3 (OK)

# Optional:
cython >=0.21                 :  0.29.22 (OK)
matplotlib >=2.0.0            :  3.4.1 (OK)
numpy >=1.7                   :  1.20.2 (OK)
pandas >=1.1.1                :  1.2.3 (OK)
scipy >=0.17.0                :  1.6.2 (OK)
sympy >=0.7.3                 :  1.7.1 (OK)
```

**Comment by furtadorebeca** [comment]
![image](https://user-images.githubusercontent.com/17478681/114252992-2b810b00-9976-11eb-8bdc-a11a57de8857.png)

**Comment by ccordoba12** [comment]
Hey @furtadorebeca, thanks for reporting. This problem was solved in pull request #15172, which will be part of our next version (5.0.1), to be released shortly.
